### PR TITLE
Addresses Issues #61 & #62

### PR DESCRIPTION
**Pull request by WhisperingChaos**
Repairs issues reported by #61 and #62.

### CODE REVIEW

**Comment by WhisperingChaos** [comment]
@jterry75 @cwilhit 

Would appreciate your review of proposed change to address polkit segmentation fault for Ubuntu 16.04 which nearly mimics the one applied by #40 for 18.04.

File: ubuntu/16.04/install.sh [hunk 1]
```diff
@@ -83,20 +83,20 @@ systemctl enable xrdp.service
...
-    }
-});
+
+# polkit policy definition language changes depending on its version. See issue #61
+if [[ "$(pkaction --version | sed -E 's/^[[:alnum:] ]*([[:digit:]]+.*)/\1/' - )" != '0.105' ]]; then
+    echo "Error: Policy rule specification probably invalid. Expected version: 0.105 detected $(pkaction --version)." >&2
+    exit 1
+fi
+
+bash -c 'cat > /etc/polkit-1/localauthority/50-local.d/45-allow-colord.pkla <<EOF
```
Comment (jterry75) [inline ubuntu/16.04/install.sh]: This is running as root so can just do `cat > /etc/polkit... <<EOF`

**Comment by jterry75** [comment]
LGTM. @cwilhit - Any issues?

**Comment by cwilhit** [comment]
LGTM

File: ubuntu/16.04/install.sh (hunk 1 above)
Comment (WhisperingChaos) [reply ubuntu/16.04/install.sh]: Will apply change requested by @jterry75.

**Comment by WhisperingChaos** [comment]
@jterry75  Rebase complete.

**Comment by jterry75** [comment]
Thanks!

**Pull request merged**